catalog/imaging/imaging-actionprovider-chip/src/test/java/org/codice/alliance/imaging/chip/actionprovider/TestImageChipActionProvider.java

```diff
@@ -64,8 +71,8 @@ public void testCanHandleInvalidMetacard() {
 
     @Test
     public void testCanHandleInvalidLocationOnMetacard() {
```

Comment:
Should also add a negative test for when the location isn't null but is an invalid location string.